### PR TITLE
[LoongArch64] amend the LA-ABI for GCStress testing.

### PR DESCRIPTION
**Pull request by shushanhf**
This is part of the issue #69705 to amend the LA's port.

This PR is to amend LA-ABI when testing under GCStress mode and also amend the tuple args.

### CODE REVIEW

**Comment by shushanhf** [comment]
Hi, @jkotas 
Could you help review this PR ?
Thanks

File: src/coreclr/vm/methodtable.cpp [hunk 1]
```diff
@@ -3338,23 +3349,33 @@ int MethodTable::GetLoongArch64PassStructInRegisterFlags(CORINFO_CLASS_HANDLE cl
...
+                        size = STRUCT_FIRST_FIELD_SIZE_IS8;
+                    }
                 }
                 else if (fieldType == ELEMENT_TYPE_CLASS)
                 {
-                    MethodTable* pMethodTable2  = pFieldStart->GetFieldTypeHandleThrowing().GetMethodTable();
+                    MethodTable* pMethodTable2;
 
-                    if (pMethodTable2->IsArray())
+                    TypeHandle th = pFieldStart->LookupFieldTypeHandle();
```
Comment (jkotas) [inline src/coreclr/vm/methodtable.cpp:3361]: `LookupFieldTypeHandle` can return null when the type was not loaded yet. I do not think it is correct to use it here.

Comment (shushanhf) [reply src/coreclr/vm/methodtable.cpp:3361]: Yes, you are right.
I had met this case.

How about adding `if (pMethodTable2 != nullptr)`

Comment (jkotas) [reply src/coreclr/vm/methodtable.cpp:3361]: > How about adding if (pMethodTable2 != nullptr) ?

That does not work either. This method has to be deterministic, always return the same output for given input.

Comment (shushanhf) [reply src/coreclr/vm/methodtable.cpp:3361]: > > How about adding if (pMethodTable2 != nullptr) ?
> 
> That does not work either. This method has to be deterministic, always return the same output for given input.

Thanks very much!

> there is another way that deleting all the `ELEMENT_TYPE_CLASS` which is being testing on my machine.

How about this ? I will update this PR liking this ?

**Review by jkotas — APPROVED** [review]

**Pull request merged**